CDC Datasets (#806)
rfordatascience/tidytuesday: data/curated/new_submission/intro.Rmd

This week we're exploring datasets that the Trump administration has purged. 

> The Trump administration is [purging Federal websites that contain information that the administration finds objectionable](https://www.npr.org/sections/shots-health-news/2025/01/31/nx-s1-5282274/trump-administration-purges-health-websites). 

Author: ```suggestion
The [Trump administration has ordered agencies to purge their websites of any references to topics such as LGBTQ+ rights](https://www.npr.org/sections/shots-health-news/2025/01/31/nx-s1-5282274/trump-administration-purges-health-websites). 

```

> This week we're exploring datasets that the Trump administration find objectionable. 

Reviewer: ```suggestion
This week we're exploring datasets that the Trump administration has purged. 

```

> The Trump administration is [purging Federal websites that contain information that the administration does not like](https://www.npr.org/sections/shots-health-news/2025/01/31/nx-s1-5282274/trump-administration-purges-health-websites). 

Reviewer: ```suggestion
The Trump administration is [purging Federal websites that contain information that the administration finds objectionable](https://www.npr.org/sections/shots-health-news/2025/01/31/nx-s1-5282274/trump-administration-purges-health-websites). 

```

Author: Your other suggestion helped me see a general path to fixing these: Focus on the action, not the motivation. We know what they did, things that have been said, etc. I'll remove cases where I talk about feelings. So I'm going to commit this change, but then I'll tweak it a bit more.

> The Trump administration is [purging Federal websites that contain information that the administration does not like](https://www.npr.org/sections/shots-health-news/2025/01/31/nx-s1-5282274/trump-administration-purges-health-websites). 

Author: ```suggestion
The [Trump administration has ordered agencies to purge their websites of any references to topics such as LGBTQ+ rights](https://www.npr.org/sections/shots-health-news/2025/01/31/nx-s1-5282274/trump-administration-purges-health-websites). 

```

An effort is underway to back up this publicly funded data before it is lost. 
This week's dataset contains metadata about [CDC datasets backed up on archive.org](https://archive.org/download/20250128-cdc-datasets).

> "The removal of HIV- and LGBTQ-related resources from the websites of the Centers for Disease Control and Prevention and other health agencies is deeply concerning and creates a dangerous gap in scientific information and data to monitor and respond to disease outbreaks," the Infectious Disease Society of America said in a statement. "Access to this information is crucial for infectious diseases and HIV health care professionals who care for people with HIV and members of the LGBTQ community and is critical to efforts to end the HIV epidemic."

- Which Bureaus and Programs have the most datasets archived in this collection?

> - Explore some of the datasets. What do you think this administration fears about this data?

Reviewer: I don't think the question "What do you think this administration fears about this data?"" is appropriate.

Author: Agreed! I'll remove or tweak in the next commit!

> - Explore some of the datasets. What do you think this administration fears about this data?

Author: ```suggestion
- Explore some of the datasets. What keywords do the datasets have in common?

```
